includes/_top.htm

```diff
@@ -150,6 +150,48 @@
 							</li>
 						</ul>
 					</li>
+					<li class="top-nav-item" role="menuitem">
+						<a href="#" class="top-nav-item-anchor" aria-haspopup="true" aria-expanded="false" aria-controls="nav-community-menu">
+							Apache Software
+						</a>
+						<ul class="top-nav-menu" aria-hidden="true" role="menu" id="nav-community-menu" title="Apache Homepage">
+							<li class="top-nav-menu-item" role="menuitem">
+								<a class="top-nav-anchor" tabindex="-1" href="https://www.apache.org/" target="_blank">
+									Apache Homepage
```

Comment:
The policy says

> This may either be a featured link in your main navigation system or a text link in your Main homepage text. A best practice is to include a short sentence or paragraph on the homepage noting that this project is an Apache project, and is part of a larger community of developers and users.

So making the 'Apache' part of the homepage a link, and including this information further down (after the "Trust and Ease of Use" section) should also be acceptable, and indeed closer to the best practice. 

Wdyt?

Reply:
That is fair.

Just to re-iterate so that we are on the same page, we want to remove the "Apache Software" section from the nav bar and instead add it to the footer of the homepage towards the bottom. That section will contain the necessary hyperlinks that are required by ASF. 

I am a bit rusty with my front-end skills but let me try to see what I can do here :)

Reply:
Yeah, that's what I was suggesting, if it's acceptable to others. @mimaison, @ijuma please chime in if you disagree.

Reply:
Agree with @tombentley 's suggestion.